src/xrEngine/Text_Console.cpp

```diff
@@ -85,7 +85,7 @@ void CTextConsole::CreateLogWnd()
     // Set the window's initial width
     RECT rc;
     SetRect(&rc, lX, lY, lWidth, lHeight);
-    // AdjustWindowRect( &rc, dwWindowStyle, FALSE );
+    // AdjustWindowRect( &rc, dwWindowStyle, false );
```

Comment:
Please, return `FALSE` for this function.
```suggestion
    // AdjustWindowRect( &rc, dwWindowStyle, FALSE );
```